src/main/resources/db/migration/V1_create_movies.sql

```diff
@@ -0,0 +1,9 @@
+DROP TABLE IF EXISTS movie CASCADE;
+CREATE TABLE movie
+(
+    id         INTEGER GENERATED BY DEFAULT AS IDENTITY,
+    legacy_id  INTEGER,
+    public_id  UUID,
```

Comment:
É importante alterar a coluna PUBLIC_ID para manter o padrão. Imporante saber se esse UUID será gerado pelo banco ou inserido pela aplicação? Se for pelo banco, abaixo o exemplo de como ficaria:

"ID_PUBLIC STRING AS UUID GENERATED BY DEFAULT,"